lib/Cake/Model/Datasource/CakeSession.php

```diff
@@ -575,7 +575,7 @@ protected static function _configureSession() {
 			$sessionConfig['cacheLimiter'] = 'must-revalidate';
 		}
 
-		if (empty($_SESSION)) {
+		if (!headers_sent() && session_status() !== \PHP_SESSION_ACTIVE  && empty($_SESSION)) {
```

Comment:
`PHP Fatal error:  Call to undefined function session_status()` on PHP: 5.3. Add check with `function_exists()`.

CakePHP 2 is namespaceless, so `\` is inconsistent here.

`578 | ERROR | Double space found`